django_project/species/views.py

```diff
@@ -16,3 +16,16 @@ def get(self, request):
             status=200,
             data=TaxonSerializer(taxon, many=True).data
         )
+
+
+class TaxonFrontPageListAPIView(APIView):
```

Comment:
Is this related to this ticket?

Reply:
Yeah, this is for ticket #669 showing national trends on landing page. Previously, the species list was hardcoded and I changed it to pull from API.